Userland/Libraries/LibCore/Process.cpp

```diff
@@ -23,6 +23,10 @@
 #    include <syscall.h>
 #elif defined(AK_OS_BSD_GENERIC) && !defined(AK_OS_SOLARIS)
 #    include <sys/sysctl.h>
+#elif defined(AK_OS_GNU_HURD)
+extern "C" {
+#    include <hurd.h>
```

Comment:
Seems to me that it's a bug in the Hurd that this header doesn't have `__BEGIN_DECLS/__END_DECLS` :)

Reply:
It is, yes. Fixing this is ok my to-do list for glibc,  but we're going to have to do this for the time being.

This is what [GDB also does](https://github.com/bminor/binutils-gdb/blob/b3a1e48614c3ae9a9be09e32183b9a57c93e1f88/gdb/gnu-nat.c#L26).